lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -52,6 +52,9 @@ static const char *headerFileTemplate =
 
 #include <stdint.h>
 
+// Glow bundle error codes
```

Comment:
Missing period at the end of a sentence/comment.

Reply:
Done here and in the other places missing periods from the auto-generated header file.